x-pack/legacy/plugins/maps/public/shared/layers/sources/ems_tms_source/ems_tms_source.js

```diff
@@ -3,16 +3,18 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
```

Comment:
I would also modify the display of this in the source-details. So modify `EMSTMSource.getImmutableProperties` to indicate this is autoselected. Maybe just add `(autoselected)` in parantheses or something , or as a new property.